packages/mirage-conduit/mirage-conduit.3.0.0/opam

```diff
@@ -15,19 +15,22 @@ build: [
 depends: [
   "ocaml" {>= "4.03.0"}
   "jbuilder" {>= "1.0+beta10"}
-  "ppx_sexp_conv" {< "v0.15"}
-  "cstruct" {>= "3.0.0"}
+  "ppx_sexp_conv" {< "v0.14.2"}
+  "cstruct" {>= "3.0.0" & < "4.0.0"}
   "mirage-types-lwt" {>= "3.0.0" & < "3.7.0"}
   "mirage-flow-lwt" {>= "1.2.0"}
+  "dns"
+  "dns-lwt"
   "mirage-dns" {>= "3.0.0" & < "4.0.0"}
-  "conduit-lwt" {< "3.0.0"}
+  "conduit" {= "1.0.0"}
+  "conduit-lwt" {= "1.0.0"}
```

Comment:
Is that necessary? Would it be only available with `1.0.0` before? I'd rather avoid strict constraints in opam-repo

Reply:
All the versions of `mirage-conduit` have been released in sync with `conduit` (but with a different version number). I think it's better to keep them constrained, especially if that help opam 2.0's solver.